spec/requests/reservations_request_spec.rb

```diff
@@ -22,19 +22,21 @@
   end
 
   describe 'POST /reservations' do
-    let(:seance) { create(:seance) }
-    let(:client) { create(:client) }
-    let(:ticket_desk) { create(:ticket_desk) }
+    let!(:seance) { create :seance }
```

Comment:
`create :seance, seance_date: DateTime.zone.now`

Reply:
Fixed -> there was no seance_date in factory